CHANGELOG.md

```diff
@@ -1,37 +1,46 @@
 This is a history of changes to clara-rules.
 
-# 0.23.0-SNAPSHOT
+# 0.25.0-SNAPSHOT
+
+
+### 0.24.0
+* uplift to cljs 1.11.132
+* uplift to clj 1.11.2
+* remove atom usage in LHS functions
+* remove redundant TestNode evaluations
+
+### 0.23.0
 * extract clara.rules.compiler/compile-test-handler from clara.rules.compiler/compile-test
 * add support for `env` inside of test expressions
 * use `.clj_kondo` extension for clj-kondo hook code for better tool compatibility (clj-kondo support now requires clj-kondo 2022.04.25 or higher)
 * Include the invalid constraint in the exception thrown at session compilation time when negations have multiple children.  See [Issue 284](https://github.com/cerner/clara-rules/issues/284).
 
-# 0.22.1
+### 0.22.1
```

Comment:
Why triple now?

Reply:
The initial were triple, and it felt like we were mixing and matching. 

I flipped all but the active to being a single `#` to accent the distinction between ongoing and released versioning.